cmd/influx/remote.go

```diff
@@ -94,16 +98,40 @@ func newRemoteDeleteCmd() cli.Command {
 				Required:    true,
 				Destination: &remoteID,
 			},
+			&cli.BoolFlag{
+				Name:        "delete-replications",
+				Usage:       "Forcefully deletes any replication queues that depend on this remote",
+				Destination: &deleteReplications,
+			},
 		),
 		Action: func(ctx *cli.Context) error {
 			api := getAPI(ctx)
+			cli := getCLI(ctx)
+			context := getContext(ctx)
 
 			client := remote.Client{
-				CLI:                  getCLI(ctx),
+				CLI:                  cli,
 				RemoteConnectionsApi: api.RemoteConnectionsApi,
 			}
 
-			return client.Delete(getContext(ctx), remoteID)
+			if deleteReplications {
+				conn, err := client.Get(context, remoteID)
```

Comment:
I don't like doing this client action here and pulling an API struct into `cmd`, but the alternative is doing replication things in `remotes` dir, or the other way around, so I think this is the best way to handle it and keep these two as separated as possible